frontend/src/components/CreateRecipeForm/CreateRecipeForm.js

```diff
@@ -42,7 +44,32 @@ const EditRecipeForm = ({ history }) => {
   const [selectedTagsWithId, setSelectedTagsWithId] = useState([]);
   const [refreshTags, setRefreshTags] = useState(true);
   const [ingredients, setIngredients] = useState(['']);
+  const [isEditing, setIsEditing] = useState(false);
 
+  const handleTitleChange = (x) => {
+    setTitle(x);
+    setIsEditing(true);
+  };
```

Comment:
Can we add some blank lines between all these function definitions, here and in the ingredient/direction files?